README.md

```diff
@@ -8,46 +9,53 @@ OpenTelekomCloud driver for docker-machine
 ### Comparing with other drivers
 
 There are two more options of docker-machine driver suitable for usage with OpenTelekomCloud:
- * [docker-machine-openstack](https://opendev.org/x/docker-machine-openstack) ― docker-machine built-in
- * [DockerMachine4OTC](https://github.com/Huawei/DockerMachineDriver4OTC) ― older OTC driver implementation by Huawei
-
-This driver is inspired by `docker-machine-openstack` and targets to provide full backward compatibility with `DockerMachineDriver4OTC`.
-
-Backward compatibility meaning, that this driver uses flags and env variable naming that differs from OpenStack convention.
-Thus, in versions `0.3+` duplicating options will be removed and all environment variables will be prefixed with `OS_`.
-
-Feature                                         | OTC (new)     | OTC (old) | Openstack
----                                             | ---           | ---       | ---
-Automated creation of required infrastructure   | **Yes**       | No        | No
-Support of `clouds.yaml` and `OS_CLOUD`         | **Yes**       | No        | No
-Support using of resource names instead of IDs  | Yes           | No        | Yes
-User data injection                             | Yes           | No        | Yes
-Floating IP pool selection                      | No            | No        | Yes
-Custom CA usage                                 | No            | No        | Yes
-Insecure mode (without TLS certificate check)   | No            | No        | Yes
-Bandwidth configuration                         | Yes           | Yes       | No
-Root volume configuration                       | Yes           | Yes       | No
-Optional usage of floating IP                   | Yes           | Yes       | No
-AK/SK auth                                      | Yes           | Yes       | No
-Server group                                    | **Yes**       | No        | No
-Security group(s)                               | Multiple      | Single    | Multiple
-Prepared k8s security group                     | Yes           | No        | No
-Usage with rancher                              | Needs setup   | Built-in  | Built-in
+
+* [docker-machine-openstack](https://opendev.org/x/docker-machine-openstack) ― docker-machine built-in
+* [DockerMachine4OTC](https://github.com/Huawei/DockerMachineDriver4OTC) ― older OTC driver implementation by Huawei
+
+This driver versioned `v0.2.x` is inspired by `docker-machine-openstack` and targets to provide full backward compatibility
```

Comment:
Both versions are inspired by `docker-machine-openstack`. But only `v0.2` aims for compatibility